starlingcaptureapi/file_util.py

```diff
@@ -53,14 +53,35 @@ def digest_sha256(self, file_path):
 
         Returns:
             the HEX-encoded SHA-256 digest of the input file
+
+        Raises:
+            any file I/O errors
         """
         hasher = sha256()
         with open(file_path, "rb") as f:
             # Parse file in blocks
-            for byte_block in iter(lambda: f.read(4096), b""):
+            for byte_block in iter(lambda: f.read(32 * 1024), b""):
```

Comment:
I would extract the magic block size number into a module-level constant, especially since it is reused.